docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -269,14 +302,38 @@ ERROR: ssh: rejected: administratively prohibited (unauthorized device)
 ```
 
 <Admonition type="tip" title="Trusted Clusters">
-It is possible to use [trusted
-clusters](../../management/admin/trustedclusters.mdx) to limit the impact of
-device mode `required`. A leaf cluster in mode `required` will enforce access to
-all of its resources, without imposing the same restrictions to the root
-cluster. Likewise, a root cluster will not enforce device trust on resources in
-leaf clusters.
+  It is possible to use [trusted
+  clusters](../../management/admin/trustedclusters.mdx) to limit the impact of
+  device mode `required`. A leaf cluster in mode `required` will enforce access
+  to all of its resources, without imposing the same restrictions to the root
+  cluster. Likewise, a root cluster will not enforce device trust on resources
+  in leaf clusters.
 </Admonition>
 
+## Locking device
+
+Similar to [session and identity locking](./locking.mdx), a device can be locked using `tctl lock`.
+
+```code
+# Find device id to lock
+$ tctl devices ls
+Asset Tag     OS    Enroll Status   Device ID
+------------ -----  ------------- ------------------------------------
+C0XXXXXXXXXX  macOS  enrolled      9cdfc0ad-64b7-4d9c-9342-50e97f418ba0
+
+# Lock device
+$ tctl lock --device 9cdfc0ad-64b7-4d9c-9342-50e97f418ba0 --ttl=12h
```

Comment:
Please update the tctl reference by adding the `--device` flag.

Reply:
Can you please point the line number again? Above command reference already uses `--device` flag?

Reply:
I was thinking of https://goteleport.com/docs/reference/cli/#tctl, but it looks like `tctl lock` is not there. I think you got it on locking.mdx, though.

Reply:
Let's add `tctl lock` to that page. Doesn't have to be in this PR, but we can at least create a new issue to track that.